server/server.go

```diff
@@ -209,16 +208,23 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 		clients:           make(map[uint32]*clientConn),
 		stopListenerCh:    make(chan struct{}, 1),
 	}
-	s.loadTLSCertificates()
+
+	tlsConfig, err := util.LoadTLSCertificates(s.cfg.Security.SSLCA, s.cfg.Security.SSLKey, s.cfg.Security.SSLCert)
+	if err != nil {
+		logutil.BgLogger().Warn("secure connection is not enabled")
```

Comment:
I think we should return the error here, but the old implement doesn't return an error either. It is weird.

Reply:
+1 we should return error here